### ISSUE TITLE
snakemake not working correctly with Mac

### ISSUE THREAD

**Issue by romainsacchi** [romainsacchi/bw-repro#4]
For some unknown reason, I cannot run the snakemake pipeline properly.
When it comes to the part of downloading and installing ecoinvent, snakemake triggers the same pipeline again.
Not sure how to solve this...

```
Creating project...
Installing biosphere...
Databases dictionary with 4 object(s):
	biosphere3
	cobalt
	dummy_bio
	dummy_db
Biosphere database already present!!! No setup is needed
Installing ecoinvent...
ecoinvent username: xxxxxx
ecoinvent password: 
logging in to ecoinvent homepage...
login successful!
downloading cutoff 3.8 ...
download finished!: /Users/romain/Library/Application Support/EcoInventDownLoader/cutoff38.7z

Extracting XML data from 19565 datasets
Creating project...
Creating project...
Creating project...
Creating project...
Creating project...

```

**Comment by tyrael147** [comment]
Could you please tell me how to use the snakemake file ? (maybe also include instructions in the README). I wanted to run it to compare it with your results but I don't know if the extracted `output.zip` folder is ok or I just don't know how to use it. 
```
> pwd
/home/glarrea/bw-repro/outputs
> ls
config  envs  resources  scripts  Snakefile
> snakemake --cores 1
Building DAG of jobs...
MissingInputException in rule run in line 1 of /home/glarrea/bw-repro/outputs/Snakefile:
Missing input files for rule run:
    output: workflow/results/results.txt
    affected files:
        workflow/config/config.json
```

**Comment by romainsacchi** [comment]
I usually do `snakemake -c1 --use-conda`

**Comment by ok-lch** [comment]
**TL;DR** jobs is an alias of cores.

I remembered we use `--jobs -j` for the hackathon and after looking at the documentation it say that its an alias for `--core -c` so it end up being the same command.
Command from hackathon: `snakemake -j 1 --use-conda`.

Documentation:
```
 --cores [N], -c [N]   Use at most N CPU cores/jobs in parallel. If N is omitted or 'all', the limit is set to the number of available CPU cores. In case of cluster/cloud execution, this argument sets the
                        maximum number of cores requested from the cluster or cloud scheduler. (See https://snakemake.readthedocs.io/en/stable/snakefiles/rules.html#resources-remote-execution for more info)This
                        number is available to rules via workflow.cores. (default: None)
```

```
  --jobs [N], -j [N]    Use at most N CPU cluster/cloud jobs in parallel. For local execution this is an alias for --cores. (default: None)
```
